src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -119,6 +120,11 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             pos += cchSingleQuoteEscape;
         }
 
+        if (translationStyle == PathTranslationStyle::MinGW)

```

Comment:
You could merge this with the following if condition to
```cpp
if (translationStyle != PathTranslationStyle::MinGW && fullPath.size() >= 2 && fullPath.at(1) == L':')
```

Reply:
Sorry for being late in getting back to this.

I thought it was clearer to early-out for the MinGW case to make it obvious that no further processing is required rather than combining the logic with the existing drive handling condition.  Readers then know there will be no additional MinGW code further on in the function.